### PR TITLE
Init the project

### PR DESCRIPTION
**Pull request by sswguo**

### CODE REVIEW

**Review by ligangty — COMMENTED** [review]
The model looks good to me as a start. But for the project itself, do we really need a multi-module project here? Maybe we can organize all these classes in different packages, but not a different modules.

And maybe we should think about to define the json schema for these models. I'll try to start a start docs for it later.

**Comment by sswguo** [comment]
> The model looks good to me as a start. But for the project itself, do we really need a multi-module project here? Maybe we can organize all these classes in different packages, but not a different modules.
> 
> And maybe we should think about to define the json schema for these models. I'll try to start a start docs for it later.

Sure, it's good to have the docs.:) And we can use different packages rather than the modules.

**Review by ligangty — APPROVED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>pom</packaging>
+
+  <parent>
+    <groupId>org.commonjava</groupId>
+    <artifactId>commonjava</artifactId>
+    <version>16</version>
+  </parent>
```
Comment (ligangty) [inline pom.xml:26]: Maybe we need a different parent pom here to consolidate common libs like quarkus.

Comment (sswguo) [reply pom.xml:26]: Do we use the different groupID for the micro services and event handlers ? I will update this after we lock that down.

Comment (ligangty) [reply pom.xml:26]: I think the "org.commonjava" for groupId is ok. For the artifact-id, how about "service-parent"? And I think we need a separate repo to hold this parent pom.
CC @ruhan1

Comment (sswguo) [reply pom.xml:26]: "service-parent" looks good.

Comment (ruhan1) [reply pom.xml:26]: service-parent looks good.

File: src/main/java/org/commonjava/event/common/IndyEvent.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package org.commonjava.event.common;
+
+import org.commonjava.event.common.EventMetadata;
+
+import java.util.UUID;
+
+public class IndyEvent
+{
+
```
Comment (ruhan1) [inline src/main/java/org/commonjava/event/common/IndyEvent.java:9]: we might need a versionId. Initially it is empty. Once we want to evolve it, set to sth like "v2, beta-1,...". The idea is we can send different versioned events to same topic. The old services can ignore events he does not know. I don't know if this is really needed. wdyt?

**Review by ruhan1 — COMMENTED** [review]
+1 for using packages instead of modules. let 's start from simple.

File: src/main/java/org/commonjava/event/common/IndyEvent.java (hunk 2 above)
Comment (sswguo) [reply src/main/java/org/commonjava/event/common/IndyEvent.java:9]: Yeah, that should be considered. I will add that later.

**Pull request merged**